package.json

```diff
@@ -57,7 +57,7 @@
     "husky": "^0.14.3",
     "jest": "^22.0.1",
     "rimraf": "^2.6.2",
-    "semantic-release": "^8.2.0",
```

Comment:
If you are jumping to v12 of `semantic-release` you will need to add `travis-deploy-once` to ensure it only runs once per build. See the [Migration Guide](https://github.com/semantic-release/semantic-release/releases/tag/v12.0.0)

Reply:
Nice catch, just added this 👍